### DISCUSSION TITLE
[Linux Kubuntu] Yubikey not visible if inserted after Keepass XC start

### DISCUSSION THREAD

**Discussion by ElucGeek** [keepassxreboot/keepassxc#8061] [Q&A] [answered]
Hello,
I use Keepass XC for years now, mainly on Windows. Last year I added a Yubikey to ma vault for more security and recently I start using Keepass XC almost daily on Kubuntu as well.
I have found a strange feature never seen on Windows. If I insert my Yubikey after starting Keepass XC, it's not visible. I have to insert it and then start Keepass XC (or just restart it once inserted).
In most cases it's a slight incoveniance but if you happen to remove the Yubikey while Keepass XC is open with vault unlocked and you made a change and want to save it, it's impossible as the Yubikey cannot be detected even if I re-insert it. You must quit without saving to restart Keepass XC with the Yubikey already present and never remove it until you saved all changes.

I'm not sure if it's related to Keepass XC or Linux/Ubuntu/KDE Plasma. I'm not sure but it might also be the case with other app, like Yubico authenticator, that need to be restart to read the Yubikey, but in Firefox I don't have the problem, I can insert the Yubikey once I got the popup asking for it.

Is it happening to others? Any clue where to look for?

**Comment by droidmonkey** [comment, marked as answer]
How do you have keepassxc installed? Have you tried the appimage or flatpak?

**Reply by ElucGeek** [reply]
It happens that I had both the Unbuntu repo version (2.6.6) and Flatpak version (2.7.1), both installed from Discover. I removed the repo version but it doesn't change the issue as I was already using the Flatpak one.

Of course @phoerious, I did press refresh many time but it's not working.

For the Yubico Authenticator (to test another app with the Yubikey) I had the Snap version and it worked, even removing and reinserting the key. But then I did try to install in parallel the Flatpak version and now both version cannot see my account, it says no account when I insert the key, even if before starting... I then uninstalled both and reinstall only the Flatpak version and it was not working. Then I uninstall it and re-installed only the Snap version like before and it works again... Really strange.

So I did try to uninstall KeepassXC and reinstall from Flatpak (there is no Snap version in my app store) and it's like before. At start it see the key, if I remove it and refresh, no key of course. Reinsert the key, refresh, still no key visible.

**Reply by phoerious** [reply]
Try installing the PPA version.

**Reply by ElucGeek** [reply]
Ok I did and now it looks like to work. Not sure to understand what is the difference but thanks!

**Comment by phoerious** [comment]
Have you tried clicking the Refresh button?
